doc/classes/CPUParticles.xml

```diff
@@ -71,134 +95,197 @@
+			Radial acceleration randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="randomness" type="float" setter="set_randomness_ratio" getter="get_randomness_ratio">
+			Emission lifetime randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="scale_amount" type="float" setter="set_param" getter="get_param">
+			Initial scale applied to each particle.
 		</member>
 		<member name="scale_amount_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's scale will vary along this [CurveTexture].
 		</member>
 		<member name="scale_amount_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Scale randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="speed_scale" type="float" setter="set_speed_scale" getter="get_speed_scale">
+			Particle system's running speed scaling ratio. Default value: [code]1[/code]. A value of [code]0[/code] can be used to pause the particles.
 		</member>
 		<member name="spread" type="float" setter="set_spread" getter="get_spread">
+			Each particle's initial direction range from [code]+spread[/code] to [code]-spread[/code] degrees. Default value: [code]45[/code].
 		</member>
 		<member name="tangential_accel" type="float" setter="set_param" getter="get_param">
+			Tangential acceleration applied to each particle. Tangential acceleration is perpendicular to the particle's velocity giving the particles a swirling motion.
 		</member>
 		<member name="tangential_accel_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's tangential acceleration will vary along this [Curve].
 		</member>
 		<member name="tangential_accel_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Tangential acceleration randomness ratio. Default value: [code]0[/code].
 		</member>
 	</members>
 	<constants>
 		<constant name="DRAW_ORDER_INDEX" value="0" enum="DrawOrder">
+			Particles are drawn in the order emitted.
 		</constant>
 		<constant name="DRAW_ORDER_LIFETIME" value="1" enum="DrawOrder">
+			Particles are drawn in order of remaining lifetime.
 		</constant>
 		<constant name="DRAW_ORDER_VIEW_DEPTH" value="2" enum="DrawOrder">
+			Particles are drawn in order of depth.
 		</constant>
 		<constant name="PARAM_INITIAL_LINEAR_VELOCITY" value="0" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set initial velocity properties.
 		</constant>
 		<constant name="PARAM_ANGULAR_VELOCITY" value="1" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set angular velocity properties.
 		</constant>
 		<constant name="PARAM_ORBIT_VELOCITY" value="2" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set orbital_velocity properties.
```

Comment:
Should not have an underscore in `orbital_velocity`.

Reply:
Ah, that came from the existing ParticlesMaterial doc. I'll fix it there too.